src/MAUI/Maui.Samples/Samples/Layers/WmsIdentify/WmsIdentify.xaml.cs

```diff
@@ -97,7 +97,7 @@ private async void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.Mau
                 }
 
                 // Show a page with the HTML content
-                await Navigation.PushAsync(new WmsIdentifyResultDisplayPage(htmlContent));
```

Comment:
WmsIdentify.xaml comment: Nitpick: may benefit from responsive layout but no issue if not.

Reply:
I made a previous comment about preferring grid for label/map samples.